sktime/forecasting/compose/_pipeline.py

```diff
@@ -166,7 +169,8 @@ class ForecastingPipeline(_Pipeline):
 
     _required_parameters = ["steps"]
     _tags = {
-        "univariate-only": False,
+        "scitype:y": "both",
+        "y_inner_mtype": ["pd.Series", "pd.DataFrame"],
```

Comment:
hm, due to the duplication: should these new tags not move to `_Pipeline`? Perhaps it's safer here though, so perhaps not.